src/pages/SubmissionConfig.test.js

```diff
@@ -1583,6 +1588,95 @@ describe('SubmissionConfig submitting a run set', () => {
     )
   })
 
+  it('error message should display on workflow launch fail, and not on success', async () => {
```

Comment:
@kpierre13 is this similar to what you had in mind for a test?

Reply:
Awesome